Makefile

```diff
@@ -114,10 +114,19 @@ test_sdk: build clean
 	cd sdk && dotnet restore --no-cache
 	cd sdk/Pulumi.Tests && dotnet test --configuration Release $(DOTNET_TEST_FILTER_FLAG)
 
+.PHONY: test_sdk_automation
+test_sdk_automation: clean
+	cd sdk && dotnet restore --no-cache
+	cd sdk/Pulumi.Automation.Tests && dotnet test --configuration Release $(DOTNET_TEST_FILTER_FLAG) -p:PulumiSdkVersion=$(SDK_VERSION)
+
 .PHONY: test_coverage
-test_coverage: test_sdk_coverage
+test_coverage: test_sdk_coverage test_sdk_automation_coverage
 
 .PHONY: test_sdk_coverage
 test_sdk_coverage: clean
 	cd sdk && dotnet restore --no-cache
-	cd sdk/Pulumi.Tests && dotnet test --configuration Release $(DOTNET_TEST_FILTER_FLAG) -p:CollectCoverage=true -p:CoverletOutputFormat=cobertura -p:CoverletOutput=../../coverage/coverage.pulumi.xml
+	cd sdk/Pulumi.Tests && dotnet test --configuration Release $(DOTNET_TEST_FILTER_FLAG) -p:CollectCoverage=true -p:CoverletOutputFormat=cobertura -p:CoverletOutput=./coverage/coverage.pulumi.xml
+
+.PHONY: test_sdk_automation_coverage
+test_sdk_automation_coverage: clean
+	cd sdk/Pulumi.Automation.Tests && dotnet test --configuration Release -p:PulumiSdkVersion=$(SDK_VERSION) -p:CollectCoverage=true -p:CoverletOutputFormat=cobertura -p:CoverletOutput=./coverage/coverage.pulumi.automation.xml
```

Comment:
To be bug for bug compatible, we need to run the restore step here as well I think.